Assets/LeapMotion/Editor/CustomEditorBase.cs

```diff
@@ -118,6 +119,19 @@ public class CustomEditorBase : Editor {
       }
     }
 
+    /// <summary>
+    /// Defer rendering of a property until the end of the inspector.  Deferred properties
+    /// are drawn in the REVERSE order they are deferred!  NOT by the order they appear in the 
```

Comment:
What's the reasoning for drawing deferred properties in the _opposite_ order in which they are deferred? My intuition was initially that deferred properties would just get added in the order they are deferred to some big "deferred" group that renders last.